cmd/kubeadm/app/cmd/join.go

```diff
@@ -426,11 +423,6 @@ func (j *joinData) Run() error {
 
 	// if the node is hosting a new control plane instance
 	if j.cfg.ControlPlane != nil {
```

Comment:
i think in this PR we need to move this whole block to the phase.
because this is about joining a new control plane.
check the text "A new etcd member was added to the local/stacked"

i.e. needs to be moved to in `controlplanejoin.go`

also Run() and the call to the function should be removed at this point.
because this is the last phase.

Reply:
L425 to L439 should be moved to runEtcdLocalPhase() in controlplanejoinphase?

Reply:
yes, we basically need to remove Run() at this point.

Reply:
or removing Run() and the call to the function works either if we dont move the if j.cfg.ControlPlane != nil {.....

Reply:
ok removing Run() and its call from here